airflow/.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: go
```

Comment:
Should this file be in the root directory?

Reply:
I would suggest to use Github Actions as we do in Airflow

Reply:
This is automatic generated configuration. As far as we can use to limit the scope of changes. In the next step, we can migrate to another tool.

Reply:
this is generated by openapi-spec, which is not useful for us since we don't use travis ci. I am thinking of adding basic build commands at the root as github action to make sure the client can build without error.

Reply:
You will still be able to do this if you use - `.openapi-generator-ignore`.